test/integration/transaction.test.ts

```diff
@@ -0,0 +1,141 @@
+import { describe, before } from 'mocha'
+import { Express } from 'express'
+import { expect } from 'chai'
+
+import createHttpServer from '../../src/server'
+import { get } from '../helper/routeHelper'
+import {
+  seed,
+  cleanup,
+  seededCapacityId,
+  nonExistentId,
+  seededTransactionId,
+  seededTransactionId2,
+  seededProposalTransactionId,
+  seededAcceptTransactionId,
+  seededMatch2Id,
+  exampleDate,
+} from '../seeds'
+import { TransactionState, TransactionApiType, TransactionType } from '../../src/models/transaction'
+
+describe('transaction', () => {
+  let app: Express
+
+  before(async function () {
+    app = await createHttpServer()
+  })
+
+  beforeEach(async function () {
+    await seed()
+  })
+
+  afterEach(async function () {
+    await cleanup()
+  })
+
+  describe('happy path', () => {
+    it('it should get a transaction from an id - 200', async () => {
+        const response = await get(app, `/transaction/${seededTransactionId}`)
+        expect(response.status).to.equal(200)
+        expect(response.body).to.deep.equal({
+          id: seededTransactionId,
+          apiType: TransactionApiType.capacity,
+          transactionType: TransactionType.creation,
+          localId: seededCapacityId,
+          state: TransactionState.submitted,
+          submittedAt: exampleDate,
+          updatedAt: exampleDate,
+        })
+      })
+
+      it('it should get all transactions - 200', async () => {
+        const response = await get(app, `/transaction/`)
+        console.log(response.body)
+        expect(response.status).to.equal(200)
+        expect(response.body).to.deep.equal([
+            {
+                id: seededTransactionId,
+                api_type: TransactionApiType.capacity,
+                transaction_type: TransactionType.creation,
+                local_id: seededCapacityId,
+                state: TransactionState.submitted,
+                created_at: exampleDate,
+                updated_at: exampleDate,
```

Comment:
`expects` are failing because I changed `TransactionResponse` to use camel case keys to match rest of the repo. The tests is checking using `snake_case`